content/40.inequality.md

```diff
@@ -1,67 +1,124 @@
+These analyses illuminate the fact that most studies that identify race as a risk factor fail to consider the correlated variable of racism (see [@doi:10.1002/ajpa.20983]).
+
+Embodiment emerged in an ecosocial and epidemiological context [@doi:10.1136/jech.2004.024562], but it is also fundamentally linked to biology.
+Stress, though an adaptive response to a perceived threat, harbors a maladaptive component: the same chemicals and hormones that promote survival of an acute threat have detrimental physiological effects when an individual remains in an aroused state (i.e., stressed) over time [@doi:10.1111/j.1749-6632.1999.tb08103.x].
+The negative effects of chronic stress are known as allostatic load [@doi:10.1111/j.1749-6632.1998.tb09546.x].
+The well-documented health risks associated with allostatic load are etiologically linked to the stress response itself via the limbic-hypothalamic-pituitary-adrenal axis.
+For example, glucocorticoids, whose levels fluctuate with stress, influence the risk of sepsis [@doi:10.1016/j.molmed.2007.05.003], and hyperactivity of the amygdala combined with metabolic shifts and inflammation may promote development of atherosclerosis, cardiovascular disease, rheumatoid arthritis, hypertension, and diabetes [@doi:10.1016/j.physbeh.2011.08.019; @doi:10/dqkn6t; @doi:10.1073/pnas.081072698; @doi:10.1001/archinte.1997.00440400111013; @doi:10.1093/rheumatology/38.11.1050].
+Additionally, chronic stress may expedite physical and mental decline associated with age [@doi:10.1176/appi.ajp.2009.09040461; @doi:10.1001/archinte.1997.00440400111013].
+Other health conditions, such as hypertension, obesity, and cancer have also been linked to allostatic load [@doi:10.1001/archinte.1997.00440400111013; @doi:10.1073/pnas.081072698].
+These health disparities render chronic stress a risk factor on a biological level.
+Additionally, the list of risks associated with allostatic load closely echoes the list of comorbidities associated with more severe COVID-19 outcomes.
+
+### COVID-19 "Risk Factors"
 
 As COVID-19 has spread into communities around the globe, it has become clear that the risks associated with this disease are not equally shared by all individuals or all communities.
-Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors that influence COVID-19 severity.
-Untangling the factors influencing COVID-19 susceptibility is a complex undertaking.
+Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors associated with COVID-19 severity.
+Untangling the factors that influence COVID-19 outcomes is a complex undertaking.
 Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
```

Comment:
I actually disagree with this change because I think it makes Wuhan and China sound like separate items in a list! Would it help to say "Italy and Wuhan, China"?

Reply:
I see your point, Yes I think "Italy and Wuhan, China" would do it.